docs/Data_Portal/Users_Guide/Developers_Guide.md

```diff
@@ -298,75 +335,80 @@ const [offset, setOffset] = useState(0);
+* `sortBy` - The fields to sort the files by
+
+Note this hook was designed to take global filters (e.g. the current cohort as `case_filters`) and local filters (the
+Repository filters).
 
-Information for a single file can be queried using the `useFileSummary` hook. This call is used in the `File Summary View` page [portal.gdc.cancer.gov/files/0b5a9e7e-8e2e-4b7a-9b7e-ff5d9c5b2b2b](https://portal.gdc.cancer.gov/files/0b5a9e7e-8e2e-4b7a-9b7e-ff5d9c5b2b2b)
+Information for a single file can be queried using the `useFileSummary` hook. This call is used in
+the `File Summary View`
+page [portal.gdc.cancer.gov/files/0b5a9e7e-8e2e-4b7a-9b7e-ff5d9c5b2b2b](https://portal.gdc.cancer.gov/files/0b5a9e7e-8e2e-4b7a-9b7e-ff5d9c5b2b2b)
 
 ```typescript
- const { data: { files } = {}, isFetching } = useGetFilesQuery({
-  filters: {
-    op: "=",
-    content: {
-      field: "file_id",
-      value: setCurrentFile,
+ const {data: {files} = {}, isFetching} = useGetFilesQuery({
+    filters: {
+        op: "=",
+        content: {
+            field: "file_id",
+            value: setCurrentFile,
+        },
     },
-  },
-  expand: [
-    "cases",
-    "cases.annotations",
-    "cases.project",
-    "cases.samples",
-    "cases.samples.portions",
-    "cases.samples.portions.analytes",
-    "cases.samples.portions.slides",
-    "cases.samples.portions.analytes.aliquots",
-    "associated_entities",
-    "analysis",
-    "analysis.input_files",
-    "analysis.metadata.read_groups",
-    "downstream_analyses",
-    "downstream_analyses.output_files",
-    "index_files",
-  ],
+    expand: [
+        "cases",
+        "cases.annotations",
+        "cases.project",
+        "cases.samples",
+        "cases.samples.portions",
+        "cases.samples.portions.analytes",
+        "cases.samples.portions.slides",
+        "cases.samples.portions.analytes.aliquots",
+        "associated_entities",
+        "analysis",
+        "analysis.input_files",
+        "analysis.metadata.read_groups",
+        "downstream_analyses",
+        "downstream_analyses.output_files",
+        "index_files",
+    ],
 });
 ```
+
 The `useFileSummary` hook takes several arguments:
-* `filters` - the filters to apply to the cases and where the file uuid is passed in
-* `expand` - the fields to expand
+
+* `filters` - The filters to apply to the cases and where the file uuid is passed in
+* `expand` - The fields to expand
 
 ## Sets: Gene, SSMS, and Case
 
-Sets are supported by the GDC API and are used to create an entity that represents a set of items as a `set_id`. Sets can be
-gene sets, SSM sets, or case sets. All GDC APIs support passing sets as a filter parameter.
-The GDC Portal provides a number of hooks for creating and querying set information.
+Sets are supported by the GDC API and are used to create an entity that represents a set of items as a `set_id`. Sets
+are either gene sets, SSM sets, or case sets. All of the GDC APIs support passing sets as a filter parameter.
```

Comment:
i wonder if we need to document (later?) the intent and set_type thing for sets.

Reply:
We should, since that is new, but maybe in another update?

Reply:
yup - another update